From b64428722322052638ff38fd335ed635aab80100 Mon Sep 17 00:00:00 2001
From: ChenXiaoSong <chenxiaosong@kylinos.cn>
Date: Sun, 7 Dec 2025 13:47:46 +0800
Subject: [PATCH 24/30] smb/client: remove useless elements from
 mapping_table_ERRDOS array

The last element in the array is no longer needed.

Signed-off-by: ChenXiaoSong <chenxiaosong@kylinos.cn>
---
 fs/smb/client/netmisc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/smb/client/netmisc.c b/fs/smb/client/netmisc.c
index d7d1b9b4abcd..553d2a33b6de 100644
--- a/fs/smb/client/netmisc.c
+++ b/fs/smb/client/netmisc.c
@@ -98,7 +98,6 @@ static struct smb_to_posix_error mapping_table_ERRDOS[] = {
 	{ERRnetlogonNotStarted, -ENOPROTOOPT},
 	{ERRsymlink, -EOPNOTSUPP},
 	{ErrTooManyLinks, -EMLINK},
-	{0, 0}
 };
 
 static unsigned int errdos_num = sizeof(mapping_table_ERRDOS) /
-- 
2.43.0

